nextflow_schema.json

```diff
@@ -117,7 +117,7 @@
                     "help_text": "Where possible we are trying to collate links and settings for standard primer sets to make it easier to run the pipeline with standard keys. See https://github.com/nf-core/configs/blob/master/conf/pipeline/viralrecon/genomes.config"
                 },
                 "primer_set_version": {
-                    "type": "integer",
+                    "type": "number",
```

Comment:
We may need to change this to `string` to support semantic versioning (`4.1.1`) or other version specifiers (e.g. `4.1b`) in the future but I'm ok with this for now